### ISSUE TITLE
syntax highlight have some problem

### ISSUE THREAD

**Issue by m00nlight** [fsharp/emacs-fsharp-mode#35]
The following record syntax seems to have syntax highlight problem in fsharp mode

``` fsharp
type Point = { x : float; y: float; z: float; }
```

But the following will work

``` fsharp
type Point = { x : float, y : float}
```

Seems like a bug of `fsharp-mode-font`?

**Comment by rneatherway** [comment]
Yes, I'm not quite sure what is going on there, but I want to give it a thorough rewrite like the indentation. See some other problems in #14.

**Comment by Gastove** [comment]
This seems to be fixed? Happy to work on it if it needs any more attention.

**Comment by rneatherway** [comment]
Closed by #114
